mesheryctl/internal/cli/root/system/update.go

```diff
@@ -122,14 +184,42 @@ var updateCmd = &cobra.Command{
 				if err != nil {
 					return err
 				}
+
+				err = utils.ApplyOperatorManifest(client, true, false)
+
+				if err != nil {
+					return err
+				}
 			}
+
+			skipUpdateFlag = true
+
 			// restart the pods in meshery namespace
 			err = restart()
 
 			if err != nil {
 				return err
 			}
 
+			utils.ViperK8s.SetConfigFile(utils.DefaultConfigPath)
+			err := utils.ViperK8s.ReadInConfig()
+			if err != nil {
+				return err
+			}
+
+			kubeCompose := &config.MesheryCtlConfig{}
+			err = utils.ViperK8s.Unmarshal(&kubeCompose)
+			if err != nil {
+				return err
+			}
+
+			utils.ViperK8s.Set("contexts."+mctlCfg.CurrentContext, currCtx)
+
+			err = utils.ViperK8s.WriteConfig()
+			if err != nil {
+				return err
+			}
+
```

Comment:
Same here. I am not sure if the intention is clear here.

Reply:
This is for the same purpose as the previous, when platform: kubernetes.